docker-compose.override.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  db:
+    platform: linux/x86_64
+    volumes:
+      - ./scripts/init-db:/docker-entrypoint-initdb.d
+    environment:
+      MYSQL_DATABASE: 'momentum'
+      MYSQL_USER: 'mom'
+      MYSQL_PASSWORD: 'password'
+      MYSQL_ROOT_PASSWORD: 'root'
+
+  api:
+    platform: linux/x86_64
+    volumes:
+      - ./scripts:/app/scripts
+      - ./server/config:/app/server/config
+      - ./server/docs:/app/server/docs
+      - ./server/public:/app/server/public
+      - ./server/src:/app/server/src
+      - ./server/test:/app/server/test
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    entrypoint: sh -c "npm install -g nodemon --unsafe-perm && nodemon server.js"
+
+  integration-tests:
+    extends: api
+    entrypoint: sh -c "npm test"
+    environment:
+      NODE_ENV: 'test'
```

Comment:
I'd name this file something more meaningful: what is `override`?

Reply:
I'm not opposed to naming it docker-compose.dev.yml but that would require running `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d` instead of just `docker-compose up -d` when developing. I'd prefer using override because of this [https://docs.docker.com/compose/extends/#example-use-case](https://docs.docker.com/compose/extends/#example-use-case)